### ISSUE TITLE
Variable storing name of pythontex-files-* directory

### ISSUE THREAD

**Issue by TheChymera** [gpoore/pythontex#82]
As mentioned in issue #67 it would be great to be able to store the `.pgf`s produced by PythonTeX in the folder keeping other pythontex files - aptly named `pythontex-files-*`. This would be especially useful because it would keep my document root cleaner.

To do this I need to add the directory name of `pythontex-files-*` to [this line](https://bitbucket.org/TheChymera/pythontex_demo/src/935bbdc6d7c9da0b7bd29efef0f1fa362b3ec509/code/functions.tex?at=master&fileviewer=file-view-default#functions.tex-27). But since this directory is named contingent on my document root, the best ways to do this are:

* Change the directory naming scheme, so that it is always the same and I can add the path statically (strangely enough, I think this is best - what is the point of naming a subdirectory after its parent in the first place?)
* Have access to the exact name from within `functions.tex`

Any recommendation which solution would eb best and how I could best implement it?

**Comment by gpoore** [comment]
Hard coding a directory name would probably be the easiest thing to do at present, or perhaps create a `fig` directory.

In the next release, I should add something like `pytex.outputdir`, so that the name of the `pythontex-files-*` location is always easily accessible.

**Comment by TheChymera** [comment]
Still, I don't see why a custom-named pythontex subdir would be a good idea for a default. Would it not be better to have a consistent pythontex structure? If anybody really needs differing path names then that is something that can be customized...

**Comment by gpoore** [comment]
It would be possible to have a single directory, say `_pythontex`. Then the temp files, instead of the directory, would need to be named after the document. The problem is that if two documents are trying to share the same  `_pythontex`, and they both try to save output there, there could be a collision. Having one directory per document also makes it easier to clean up after a document that is no longer needed.

It really could work either way. I believe I had a single directory early in development, but then had some collision issues, and switched to the current approach.
